.travis.yml

```diff
@@ -1,7 +1,12 @@
 language: node_js
 node_js:
-- stable
-script: npm run lint && npm run build
+  - stable
+before_script:
+  - npm run serve &
+script:
+  - npm run lint && npm run build
+  - nightwatch --env chrome
```

Comment:
As far as I know, if you use this list syntax, it will run the next item even if the previous one failed. -> We should not run the tests if build failed.